collect_app/src/main/java/org/odk/collect/android/tasks/SaveToDiskTask.java

```diff
@@ -239,6 +241,14 @@ public static File savepointFile(File instancePath) {
         return new File(tempDir, instancePath.getName() + ".save");
     }
 
+    /**
+     * Return the name of the savepoint formIndex file for a given instance.
+     */
+    public static File savepointFormIndexIndexFile(File instancePath) {
```

Comment:
Looks like an extra `Index`. The name should ideally capture what this does. `getFormIndexFile`?

The comment is also inaccurate since this returns the `File` itself. You can also pass in `String instancePathName` rather than the file. Seems it might go better in `SaveFormIndexTask`.

Reply:
I realize that this is by analogy to `savepointFile` but that one didn't really make sense either.

Reply:
I fixed naming and comments.